test/__tests__/data/attentive_tag_output.json

```diff
@@ -254,8 +254,45 @@
     "files": {}
   },
   {
-    "statusCode": 400,
-    "error": "[Attentive Tag] :: Either email or phone is required"
```

Comment:
Earlier this was a 400 error but now we are getting a response. Is this expected?

Reply:
Yes

Reply:
So, we have external identifiers also apart from `email` and `mobile`. Is this mentioned in the doc or needs any change?

Reply:
This wasn't mentioned in the doc. But, we can enhance the doc a bit.